MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -299,50 +299,10 @@ private StreamInfo BuildAudioItem(MediaSourceInfo item, AudioOptions options)
 
             if (directPlayMethods.Any())
             {
-                string audioCodec = audioStream?.Codec;
+                var conditions = GetConditionsForCodec(audioStream?.Codec, item.Container, options.Profile.CodecProfiles, CodecType.Audio, item, audioStream);
+                var failedConditions = FindFailedConditions(conditions, CodecType.Audio, item, audioStream);
```

Comment:
You don't seem to use the result of `FindFailedConditions`, so wouldn't it make more sense to change the return type to "bool" like "AllConditionsSatisfied"?